docs/pages/versions/unversioned/sdk/apple-authentication.mdx

```diff
@@ -58,47 +67,73 @@ Running [EAS Build](/build/introduction) locally will use [iOS capabilities sign
 
 ## Usage
 
-```js
+<SnackInline label="Apple Authentication Usage" dependencies={['expo-apple-authentication']} platforms={['ios']}>
+
+```jsx
 import * as AppleAuthentication from 'expo-apple-authentication';
+import { View, StyleSheet } from 'react-native';
 
-function YourComponent() {
+export default function App() {
   return (
-    <AppleAuthentication.AppleAuthenticationButton
-      buttonType={AppleAuthentication.AppleAuthenticationButtonType.SIGN_IN}
-      buttonStyle={AppleAuthentication.AppleAuthenticationButtonStyle.BLACK}
-      cornerRadius={5}
-      style={{ width: 200, height: 44 }}
-      onPress={async () => {
-        try {
-          const credential = await AppleAuthentication.signInAsync({
-            requestedScopes: [
-              AppleAuthentication.AppleAuthenticationScope.FULL_NAME,
-              AppleAuthentication.AppleAuthenticationScope.EMAIL,
-            ],
-          });
-          // signed in
-        } catch (e) {
-          if (e.code === 'ERR_CANCELED') {
-            // handle that the user canceled the sign-in flow
-          } else {
-            // handle other errors
+    <View style={styles.container}>
+      <AppleAuthentication.AppleAuthenticationButton
+        buttonType={AppleAuthentication.AppleAuthenticationButtonType.SIGN_IN}
+        buttonStyle={AppleAuthentication.AppleAuthenticationButtonStyle.BLACK}
+        cornerRadius={5}
+        style={styles.button}
+        onPress={async () => {
+          try {
+            const credential = await AppleAuthentication.signInAsync({
+              requestedScopes: [
+                AppleAuthentication.AppleAuthenticationScope.FULL_NAME,
+                AppleAuthentication.AppleAuthenticationScope.EMAIL,
+              ],
+            });
+            // signed in
+          } catch (e) {
+            if (e.code === 'ERR_CANCELED') {
+              // handle that the user canceled the sign-in flow
+            } else {
+              // handle other errors
+            }
           }
-        }
-      }}
-    />
+        }}
+      />
+    </View>
   );
 }
+
+/* @hide const styles = StyleSheet.create({ ... }); */
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  button: { 
+    width: 200, 
+    height: 44
+  }
+});
+/* @end */
 ```
 
+</SnackInline>
+
 ## Development and Testing
 
-You can test this library in development in Expo Go on iOS without following any of the instructions above; however, you'll need to do this setup to use Apple Authentication in your standalone app. When you sign into Expo Go, the identifiers and values you receive will likely be different than what you'll receive in standalone apps.
+You can test this library in development in Expo Go on iOS without following any of the instructions above.
+However, you'll need to do this setup to use Apple Authentication in your standalone app.
```

Comment:
```suggestion
However, you'll need to do this setup to use Apple Authentication in your standalone app.
```

Un related to this PR since the text was already there, it is unclear what "this setup" references here to? Are we talking about the config plugins? I think, we can be explicit here that one has to use config plugins before creating app binary/standalone app.

Reply:
I'm either not 100% sure what this sentence is referring to, but from the Git history, it looks like it was related to the initial setup steps, when this package has been added in #5421.

Reply:
Thanks for digging in! Looks these steps were moved to "Classic builds" in [this PR](https://github.com/expo/expo/pull/15015/files). I think we can safely re-phrase it to:
`However, you'll need to add the config plugin if you are using EAS Build.`

Reply:
This will make the instructions clear even if the EAS Build is being used for internal distribution.

Reply:
I have replaced the sentence, as suggested. Thank you for catching that! 👍